docs/extend/index.md

```diff
@@ -231,3 +233,63 @@ in subdirectory `rootfs`.
 After that the plugin `<plugin-name>` will show up in `docker plugin ls`.
 Plugins can be pushed to remote registries with
 `docker plugin push <plugin-name>`.
+
+
+## Debugging plugins
+
+Stdout of a plugin is redirected to dockerd logs. Such entries have a
+`plugin=<ID>` suffix. Here are a few examples of commands for pluginID
+`f52a3df433b9aceee436eaada0752f5797aab1de47e5485f1690a073b860ff62` and their
+corresponding log entries in the docker daemon logs.
+
+```bash
+$ docker plugin install tiborvass/sample-volume-plugins
+```
+
+```json
```

Comment:
Tricky though, because the highlighter on docs.docker.com is different from the one at GitHub. Generally, we combine the `bash` block with the output, but put a blank line between the command that's run and the output (to make clearer where the output starts)